src/main/java/me/earth/earthhack/impl/modules/render/popchams/PopChams.java

```diff
@@ -151,5 +160,13 @@ public double getZ() {
         public ModelPlayer getModel() {
             return model;
         }
+
+        private static ModelBiped.ArmPose getArmPose(EntityPlayer player, ItemStack stack) {
+            if (stack.isEmpty()) return ModelBiped.ArmPose.EMPTY;
```

Comment:
I know its annoying but when you are here for the System.out call anyways: can you always use brackets in if-statements? I personally just cant deal with bracket-less if statements, I got many bugs from those. This does not really block the PR, since this method is fairly short but some brackets would be nice to have.